student.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'person'
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] **[OPTIONAL]** This is only optional because it is out of the scope of this project but it is important. The `parent_permission` argument uses colon notation so it is important to define it correctly in the super parameter.

  ```
  super(age, name, parent_permission: parent_permission)
  ```